spec/iterator_spec.rb

```diff
@@ -0,0 +1,220 @@
+require './iterators.rb'
+describe 'Iterators' do
+  let(:number) { [1, 2, 3, 4, 2] }
+  let(:string) { %w[cat bear rat] }
+  context '#my_all?' do
+    it 'Return true when no block is given' do
+      expect(number.my_all?).to eq true
+    end
```

Comment:
Include test for when no block is given and there is at least one element that is false or nil. In that case, it should return false

Reply:
**Noted thanks**